.github/workflows/scan-docker-images-skip.yml

```diff
@@ -1,14 +0,0 @@
-name: "Scan docker images from the allowed docker images list"
-
-on:
-  pull_request:
-    paths:
```

Comment:
I think we'll still need the skip job, because otherwise, there is no job with that name that matches the requirements on a PR. Remember?